resources/ory/charts/oathkeeper/templates/_helpers.tpl

```diff
@@ -35,6 +35,7 @@ Create chart name and version as used by the chart label.
 Common labels
 */}}
 {{- define "oathkeeper.labels" -}}
+app: {{ .Chart.Name }}
```

Comment:
This should be `{ include "oathkeeper.name" . }}`

Reply:
get it and I will change it.